CHANGELOG.md

```diff
@@ -1,3 +1,94 @@
+### 2.0.5 (2017-01-11)
+
+
+#### Bug Fixes
+
+* Do not use setState to save timer instance. ([3709fdb6](https://github.com/ubilabs/react-geosuggest/commit/3709fdb6f8d7d8e7133c153398be87268b445a7a))
+
+
+#### Features
+
+* add blur() ([d16c6105](https://github.com/ubilabs/react-geosuggest/commit/d16c6105557bcca8dabc160927a448d86f06ffbe))
+
+
+### 2.0.4 (2017-01-11)
+
+
+#### Bug Fixes
+
+* Do not use setState to save timer instance. ([3709fdb6](https://github.com/ubilabs/react-geosuggest/commit/3709fdb6f8d7d8e7133c153398be87268b445a7a))
+
+
+#### Features
+
+* add blur() ([d16c6105](https://github.com/ubilabs/react-geosuggest/commit/d16c6105557bcca8dabc160927a448d86f06ffbe))
+
+
+### 2.0.3 (2017-01-11)
+
+
+#### Bug Fixes
+
+* Do not use setState to save timer instance. ([3709fdb6](https://github.com/ubilabs/react-geosuggest/commit/3709fdb6f8d7d8e7133c153398be87268b445a7a))
+
+
+#### Features
+
+* add blur() ([d16c6105](https://github.com/ubilabs/react-geosuggest/commit/d16c6105557bcca8dabc160927a448d86f06ffbe))
+
+
+### 2.0.3 (2017-01-11)
+
+
+#### Bug Fixes
+
+* Do not use setState to save timer instance. ([3709fdb6](https://github.com/ubilabs/react-geosuggest/commit/3709fdb6f8d7d8e7133c153398be87268b445a7a))
+
+
+#### Features
+
+* add blur() ([d16c6105](https://github.com/ubilabs/react-geosuggest/commit/d16c6105557bcca8dabc160927a448d86f06ffbe))
+
+
+### 2.0.2 (2017-01-03)
+
+
+#### Bug Fixes
+
+* Do not use setState to save timer instance. ([3709fdb6](https://github.com/ubilabs/react-geosuggest/commit/3709fdb6f8d7d8e7133c153398be87268b445a7a))
+
+
+#### Features
+
+* add blur() ([d16c6105](https://github.com/ubilabs/react-geosuggest/commit/d16c6105557bcca8dabc160927a448d86f06ffbe))
+
+
+### 2.0.1 (2017-01-03)
+
+
+#### Bug Fixes
+
+* Do not use setState to save timer instance. ([3709fdb6](https://github.com/ubilabs/react-geosuggest/commit/3709fdb6f8d7d8e7133c153398be87268b445a7a))
+
+
+#### Features
+
+* add blur() ([d16c6105](https://github.com/ubilabs/react-geosuggest/commit/d16c6105557bcca8dabc160927a448d86f06ffbe))
+
+
+### 2.0.1 (2017-01-03)
+
+
+#### Bug Fixes
+
+* Do not use setState to save timer instance. ([3709fdb6](https://github.com/ubilabs/react-geosuggest/commit/3709fdb6f8d7d8e7133c153398be87268b445a7a))
+
+
+#### Features
+
+* add blur() ([d16c6105](https://github.com/ubilabs/react-geosuggest/commit/d16c6105557bcca8dabc160927a448d86f06ffbe))
+
+
```

Comment:
Please do not build the project for a PR. The release and build is done on the master branch of the project.